src/Spryker/Zed/AppPayment/Persistence/Propel/Schema/spy_app_payment.schema.xml

```diff
@@ -72,4 +72,22 @@
             <index-column name="order_item_ids" size="255"/>
         </index>
     </table>
+
```

Comment:
It is not set as required so it is already optional. The unique index is still valid IMO as it can exist only one of null.transactionId and orderReference.TransactionId.